apps/theming/lib/Themes/Ionos/LightTheme.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
thought: we could broaden this by simply calling it a eg. `nc-ionos-theme` and handle native color modes via css inside the file. By doing so, we would also enable the legacy theme based Header.